Jar/src/velma/task/Task.java

```diff
@@ -0,0 +1,52 @@
+package velma.task;
+
+public class Task {
```

Comment:
Is there a reason you chose to go for a concrete class instead of an abstract class?